src/components/molecules/TemplateSubmitted/TemplateSubmitted.tsx

```diff
@@ -55,6 +55,7 @@ export const TemplateSubmitted = ({
           <Link
             id='servicenow-link'
             href='https://nhsdigitallive.service-now.com/nhs_digital?id=sc_cat_item&sys_id=6208dbce1be759102eee65b9bd4bcbf5'
+            target='_blank'
```

Comment:
Convention for links that open in a new tab usually have (opens in a new tab) in the link text. Might want to confirm with UCD if they want it.

Reply:
Good spot - Updated to
![Screenshot 2024-11-12 at 14 32 28](https://github.com/user-attachments/assets/5e4cd15b-e05f-4073-8f8f-51de4ebab011)

After confirming with UCD